css/index.css

```
@@ -1,3 +1,382 @@
+        padding: 0;
+        width: 100%;
+        border-bottom: 1px solid black;
+    }
+}
+
+nav {
+    display: flex;
+	justify-content: space-around;
+	width: 800px;
+	margin: 20px auto;
+	flex-direction: row;
+	align-items: center; 
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+
+    nav {
+        width: 100%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    
+    nav {
+        flex-direction: column;
+        justify-content: space-evenly;
+        margin-top: 20px;
+        margin-bottom: 0;
+        /* padding: 30px 0%; */
+        width: 100%;
+        text-align: center;
+
+    }
+}
+
+nav a {
+    padding: 20px auto;
+	width: 80px;
+	color: #69686e;
+    text-decoration: none;
+}
+
+@media ( max-width: 500px ) {
+    nav a {
+        padding: 20px 0;
+        width: 100%;
+        color: grey;
+        font-size: 20px;
+        border-top: 1px solid black;
+    }
+}
+
+.logo-link {
+	 margin: 0;
+}
+
+header {
+	display: flex;
+	padding-bottom: 30px;
+	border-bottom: 2px solid black;
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    header img {
+        display: none;
+    }
+}
+
+@media ( max-width: 500px ) {
+
+    header {
+        justify-content: center;
+    }
+
+    header img {
+        display: none;
+    }
+
+}
+
+header h1 {
+	padding: 0px;
+	margin: 80px;
+	margin-bottom: 0;
+	font-size: 60px;
+	text-align: center;
+}
+
+header button {
+	margin: 20px;
+    padding: 10px 20px;
+    width: 50%;
+	border: 1px solid black;
+	background-color: white;
+}
+
+.billboard {
+	display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+	width: 400px;
+}
+
+@media (min-width: 501px) and (max-width: 800px) {
+    .billboard {
+        width: 50%;
+        margin: 0 25%;
+    }
+}
+
+@media ( max-width: 500px ) {
+    
+}
+
+.header-img {
+	padding-bottom: 30px;
+	width: 400px;
+}
+
+.header-img.services-img {
+	width: 100%;
+}
+
+@media ( max-width: 500px ) {
+    .header-img.services-img {
+
+    }
+}
+
+button {
+	margin: 20px 20px;
+	padding: 10px 10px;
+	border: 1px solid black;
+	background-color: white;
+}
+
+@media ( max-width: 500px ) {
+    header h1 {
+        margin-top: 40px;
+        font-size: 70px;
+    }
+
+    button {
+
+    }
+}
+
+
+.middle-img {
+	margin: 10px 0;
+	width: 100%;
+	 
+}
+
+/* @media ( max-width: 500px ) {
+    .middle-img {
+        padding: 5px;
+    }
+} */
+
+.content {
+	display: flex;
+	justify-content: space-between;
+	align-items: flex-start;
+	padding: 10px;
+}
+
+@media ( max-width: 500px ) {
+    .content {
+        flex-direction: column;
+        padding: 10px 0;
+    }
+}
+
+.content.services {
+	flex-wrap: wrap;
+	margin: 20px 0;
+}
+
+@media ( min-width: 501px ) and ( max-width: 800px ) {
+    .content.services {
+        justify-content: space-evenly;
+    }
+}
+
+.description {
+	display: block;
+}
+
+.content-block {
+	padding: 20px;
+	
+}
+
+@media ( max-width: 500px ) {
+    .content-block {
```

```suggestion
    .content-block {
    	margin-top: 20px;
    }
```
Remove the padding properties, with this change and the .container change at media 500px, this will make your mobile view responsive with no scroll bars on the horizontal axis of the browser